app/uk/gov/hmrc/exports/services/reversemapping/declaration/parties/DeclarationHolderParser.scala

```diff
@@ -37,6 +36,6 @@ class DeclarationHolderParser extends DeclarationXmlParser[Seq[DeclarationHolder
     val id = (inputXml \ ID).toStringOption
     val catCode = (inputXml \ CategoryCode).toStringOption
 
-    DeclarationHolder(catCode, id)
+    DeclarationHolder(catCode, id, id.map(_ => EoriSource.OtherEori))
```

Comment:
I am wondering if this is enough to use single value for all parsed decs. I don't know enough details, if there would be a problem when EORI belonged to the other category?

Reply:
No because there is nothing to stop a user select 'OtherEori' in the UI and then enter their own EORI number. The whole ticket's purpose is to save the user from having to type in their own EORI, but if they insist on typing it in we are not going to try and stop then :slightly_smiling_face:

Reply:
...also I did discuss this approach with Beth beforehand and she was happy with it :wink: